### PR TITLE
{bio}[foss/2020a] turbinesFoam v20220516

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/smoors/b963cd26ebffae4b6f927f6a19ef253e for a full test report.

File: easybuild/easyconfigs/t/turbinesFoam/turbinesFoam-20220516-foss-2020a-OpenFOAM-8.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+cmds_map = [('.*', 'source "$FOAM_BASH" && FOAM_USER_LIBBIN="$PWD/lib" ./Allwmake')]
+
+files_to_copy = ['lib']
+
+sanity_check_paths = {
+    'files': ['lib/libturbinesFoam.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
+moduleclass = 'bio'
```
Comment (verdurin) [inline easybuild/easyconfigs/t/turbinesFoam/turbinesFoam-20220516-foss-2020a-OpenFOAM-8.eb:35]: @smoors is this really `bio`?

Comment (smoors) [reply easybuild/easyconfigs/t/turbinesFoam/turbinesFoam-20220516-foss-2020a-OpenFOAM-8.eb]: good catch, fixed.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/232cc984aa201be17bc6df1dbc8b1e68 for a full test report.

**Pull request merged**